File: DollarToInr/Convert.go
/*
	Author : Pakshal Jain
	Date : 15/12/2022
	Program : To Convert Entered Money in Dollar To Inr 
*/

//Required Libraries
package main
import "fmt"

//Function To Convert Money
func Convert(No int) int {
	var ans int = 0;

	//Handeling Condition if Entered Number is Negative
	if(No < 0) {
		No = -No;
	}

	//Logic
	ans = No * 70;

	return ans;
}

///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Main Function
func main() {
	var No int = 0;
	var iret int = 0;
	
	fmt.Println("Jay ganesh...");

	//Taking Input From user
	fmt.Println("Enter Money in Dollar");
	fmt.Scan(&No);

	//Call To Convert Function
	iret = Convert(No);

	//Printing Obtained Output
	fmt.Println("Money After Conversion from Dollar To Inr is :",iret);
}

File: DollarToInr/Convert.py
"""
    Auhtor : Pakshal Jain
    Date : 15/12/2022
    Program : To Convert Entered Money From Dollar To Inr
"""

#################################################################################################################################3

#Function To Convert Money
def Convert(No) :
    ans = 0;

    #Handeling Condition if Entered Number is Negative
    if(No < 0) :
        No = -No;

    #Logic
    ans = No * 70;

    return ans;

##############################################################################################################################

#Main Function
def main() :
    No = 0;
    iret = 0;

    print("Jay Ganesh...");

    #Taking input From User
    print("Enter Money in Dollar");
    No = int(input());

    #Call To Convert Function
    iret = Convert(No);

    #Printing Obtained Output
    print("Mone Converted From Dollar To Inr is :",iret);

##########################################################################################################################################

#Call Back Method To Call Main Function
if __name__=="__main__" :
    #Call To Main Function
    main();